tests/jballerina-unit-test/src/test/resources/test-src/documentation/deprecated_annotation_project/main.bal

```diff
@@ -93,6 +93,16 @@ public type Bank record {|
     CountryCode countryCode;
 |};
 
+# Employee record
+#
+# + age - age Parameter Description
+public type Employee record {|
+    # # Deprecated
+    @deprecated
```

Comment:
Can we add a test here also with non-deprecated docs before the deprecated docs?

Reply:
```
# Employee record
#
# + age - age Parameter Description
public type Employee record {|
    # This is the description of the `Employee`'s `name` field.
    # # Deprecated
    @deprecated
    string name;
    int age;
|};
``` 
Is this correct?

Reply:
Yes.